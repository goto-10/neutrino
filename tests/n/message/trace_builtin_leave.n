import $core;

def $fail_builtin() => (@core:Array.new(1))[100];
do $fail_builtin();
