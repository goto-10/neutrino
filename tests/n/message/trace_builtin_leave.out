Error: %<condition: Signal(escape)>
Info: --- backtrace ---
- leave.out_of_bounds(100)
- [null][100]
- :fail_builtin()
